### ISSUE TITLE
Load contacts

### ISSUE THREAD

**Issue by arreche** [arreche/partners#12]
Use [Redux's Middleware](http://redux.js.org/docs/advanced/Middleware.html) to load contacts asynchronously.

**Comment by arreche** [comment]
@yeraycaballero Please share your local branch so can help you with the implementation.

**Comment by yeraycaballero** [comment]
Yeah, sorry about that, the branch fetch-contacts is on github now and I raise a pull request
